demo/eth_verifier/Makefile

```diff
@@ -17,6 +17,17 @@ deploy_verifier:
 	PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
 	forge script script/Deploy.s.sol:Deploy --via-ir --broadcast --rpc-url http://localhost:8545
 
-run_verifier:
+query:
+	./query.sh | xargs -0 ./state_serializer > state.mpk
```

Comment:
the state_serializer file should be compiled in the Makefile as part of the execution

Reply:
done!